### PR TITLE
fix harbor using with pspNames

### PR DESCRIPTION
**Pull request by MinerYang**
Signed-off-by: yminer <yminer@vmware.com>

## What this PR does / why we need it
 we found a issue need to be fixed when we are using PSP which should switch origin Role/Role Binding into ClusterRole / ClusterRoleBinding instead. 
## Which issue(s) this PR fixes

Fixes: #

### CODE REVIEW

File: addons/packages/harbor/2.5.3/bundle/config/globals.star [hunk 1]
```diff
@@ -71,5 +71,9 @@ def get_resource_names_for_role(names):
   end
 end
 
+def setClusterRoleName():
```
Comment (zyyw) [inline addons/packages/harbor/2.5.3/bundle/config/globals.star:74]: the implementation of this function doesn't actually `set` any value but return a value. Maybe it should be `getXXX()`?

Comment (MinerYang) [reply addons/packages/harbor/2.5.3/bundle/config/globals.star:74]: I actually using get at the beginning... How about we use `assembleClusterRoleName` or `concatClusterRoleName`.
@heww

File: addons/packages/harbor/2.5.3/bundle/config/overlay/add-rbac.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  - podsecuritypolicies
+  verbs:
+  - use
+
+#@ if/end values.pspNames:
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: #@ setClusterRoleName()
```
Comment (heww) [inline addons/packages/harbor/2.5.3/bundle/config/overlay/add-rbac.yaml:25]: `setClusterRoleName` => getClusterRoleBindingName

File: addons/packages/harbor/2.5.3/bundle/config/globals.star (hunk 1 above)
Comment (heww) [inline addons/packages/harbor/2.5.3/bundle/config/globals.star:74]: setClusterRoleName => getClusterRoleName

File: addons/packages/harbor/2.5.3/bundle/config/overlay/add-rbac.yaml (hunk 2 above)
Comment (heww) [reply addons/packages/harbor/2.5.3/bundle/config/overlay/add-rbac.yaml:25]: We can add two functions in the globals.star.

One is getClusterRoleName, another is getClusterRoleBindingName

Comment (MinerYang) [reply addons/packages/harbor/2.5.3/bundle/config/overlay/add-rbac.yaml:25]: I am using this function to concat both `ClusterRole`  name and `ClusterRoleBinding` name

**Review by heww — APPROVED** [review]

**Review by zyyw — APPROVED** [review]

**Pull request merged**